sass/components/_ciq-crosshair.scss

```diff
@@ -38,7 +38,8 @@
             color: themed('CrosshairText');
             background: themed('CrosshairBg');
         }
-        display: flex;
+        display: none;
+        // we make it display flex in js
```

Comment:
@nazanin-binary Regarding the changes in crosshair display, IMO showing and hiding the crosshair by setting `display` to `none` has some rendering cost. 
@aminroosta Would you please check it?

Reply:
@nazanin-binary changing `display` will force browser to re calculate the css layout.
It's expensive and it's better to avoid.

The alternative is to set `visibility: hidden` or `translate: transform  (-5000px, -5000px)` or `opacity: 0` depending on what we are trying to achieve.

Reply:
I removed all css changes